installing/installing_openstack/installing-openstack-installer-custom.adoc

```diff
@@ -0,0 +1,61 @@
+[id="installing-openstack-installer-custom"]
+= Installing a cluster on OpenStack with customizations
+include::modules/common-attributes.adoc[]
+:context: installing-openstack-installer-custom
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a customized cluster on
+OpenStack. To customize the installation, modify parameters in the `install-config.yaml` before you install the cluster.
+
+.Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+
+* Have access to an OpenStack administrator's account
+
+* In order to run the {product-title} installer in OpenStack, your quota should meet these requirements:
++
+.Minimum recommended resources for a default OpenShift cluster on OpenStack
+[options="header"]
+|================================
+|Resource              | Value
+|Floating IP addresses | 2
+|Routers               | 1
+|Subnets               | 1
+|RAM                   | 112 GB
+|vCPUs                 | 28
+|Volume storage        | 175 GB
+|Instances             | 7
+|Security groups       | 3
+|Security group rules  | 60
+|================================
```

Comment:
Could you please add the quota for Swift containers, objects and space? We've added it to the upstream docs very recently:

https://github.com/openshift/installer/tree/master/docs/user/openstack#openstack-requirements

There's a bigger issue we need to address where this Swift quota is for the installation only. If you want to run workloads using the image registry on the deployed cluster (such as building the container images), the numbers need to be much higher -- but we don't currently know how much.

We can't even come up with a number (depends on how many images etc. you'll be building) but I'd like to have something like: for every OCP pod, you need x objects and y MB.

Reply:
@tomassedovic Yes to that first part.

As to the rest, I can absolutely add guidance like that, too--just need to have some figures.